<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>hhw的五子棋</string>
  </property>
  <property name="windowIcon">
   <iconset resource="media.qrc">
    <normaloff>:/media/black.png</normaloff>:/media/black.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="board" native="true">
      <property name="minimumSize">
       <size>
        <width>550</width>
        <height>550</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>550</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 16pt &quot;楷体&quot;;</string>
      </property>
      <property name="title">
       <string>棋盘信息</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>黑方玩家:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>白方玩家:</string>
       </property>
      </widget>
      <widget class="QLabel" name="blackPlayerLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="whitePlayerLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="blackChessPtr">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/media/black.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="whiteChessPtr">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/media/white.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="reStart">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>370</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>新开一局</string>
       </property>
      </widget>
      <widget class="QPushButton" name="choosePlayer">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>430</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>更换选手</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>490</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>结束游戏</string>
       </property>
      </widget>
      <widget class="QLabel" name="gameProgress">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>游戏未开始</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>游戏进程：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="blackStepTimeLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>本次时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="blackTotalTimeLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>总时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="whiteStepTimeLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="whiteTotalTimeLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>本次时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>总时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="blackPerStepLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>步均时间:</string>
       </property>
      </widget>
      <widget class="QLabel" name="whitePerStepLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>220</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>步均时间:</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="media.qrc"/>
 </resources>
 <connections/>
</ui>
